### ISSUE TITLE
Error on OS X installation - /usr/local/lib/liblua.a: No such file or directory

### ISSUE THREAD

**Issue by solidfoxrock** [aerospike/aerospike-client-php#16]
Hi, 

I got this error on my first installation try on OS X:

```
  ls: /usr/local/lib/liblua.a: No such file or directory
  ld: library not found for -llua
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  make: *** [aerospike.la] Error 1
```

Changing the following line in `build.sh` solved my case (I installed lua with homebrew):

```
- ec=`ls /usr/local/lib/liblua.a`
+ ec=`ls /usr/local/opt/lua/lib/liblua.dylib`
```

Best,
Kien

**Comment by rbotzer** [comment]
Great, I'll double check it and commit. Which version of the brew package is it?

**Comment by solidfoxrock** [comment]
I'm using Homebrew 0.9.5, lua is 5.2.3

**Comment by rbotzer** [comment]
We have a fix in progress that have to consider first, but I wanted to note that we need Lua 5.1.4 as a prerequisite. We do not work with Lua 5.2 due to language incompatibilities.

**Comment by solidfoxrock** [comment]
Thanks for your note, Ronen

**Comment by rbotzer** [comment]
Just an update, we're close to shipping a version of the C client that includes the Lua libraries in it, so the pre-requisite will go away.

**Comment by b-a-t** [comment]
@rbotzer Not sure it's a good idea in an environments like Debian, for example. I'd rather have one single point of (failure) upgrade. Would it be possible to make such a bundling optional? At least, when building from source?

**Comment by rbotzer** [comment]
Thanks for the input, Timur. I believe we'll allow the paths to be overwritten. It's been a sore point for people to install Lua, especially a slightly off version of it that we expect (5.1 instead of 5.2).

**Comment by b-a-t** [comment]
@rbotzer having nginx, redis and aerospike Lua-enabled on the same cluster of machines makes me feel that having one Lua to rule them all is more consistent :) But I could be wrong :)

**Comment by rbotzer** [comment]
Sounds great, in theory, unless they all require different versions :grimacing: Oh, the humanity!

We will leave an ability to use a different Lua module than the one packaged with the client, but I assume most users won't use it.

**Comment by b-a-t** [comment]
As soon as there is an option it's perfect!

**Comment by rbotzer** [comment]
Fixed in release >= 3.3.10.
